slu-scripts/auth-proof/encrypt-data.ts

```diff
@@ -0,0 +1,17 @@
+import * as vpuf from '../vpuf/vpuf';
+
+export function encryptData() {
+	const keyFilePath: string | undefined = process.env.KEY;
+	const inputData: string | undefined = process.env.INPUT;
+	const destination: string | undefined = process.env.DEST;
+	try {
```

Comment:
Please remove your try catch from the function this should be handled by the calling tool or script